test/WebJobs.Script.Tests/Rpc/WebHostLanguageWorkerChannelManagerTests.cs

```diff
@@ -59,7 +59,7 @@ public WebHostLanguageWorkerChannelManagerTests()
             _languageWorkerProcessFactory = new Mock<ILanguageWorkerProcessFactory>();
             _languageWorkerProcessFactory.Setup(m => m.CreateLanguageWorkerProcess(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>())).Returns(_languageWorkerProcess.Object);
 
-            _languageWorkerChannelFactory = new TestLanguageWorkerChannelFactory(_eventManager, null, _scriptRootPath);
+            _languageWorkerChannelFactory = new TestLanguageWorkerChannelFactory(_eventManager, new TestLogger("WebHostLanguageWorkerChannelManagerTests"), _scriptRootPath);
```

Comment:
move creating testLogger to constructor. Are you using this to verify logs? if not undo

Reply:
I think we need to do this to, otherwise tests fail with a null exception here! In the constructor of CreateTestLanguage, it takes in an "IMetricsLogger" and not a "TestLogger"

Reply:
Add tests that verify specialization logs, see here for example on verifying test logs: azure-functions-host\test\WebJobs.Script.Tests\Rpc\LanguageWorkerChannelTests.cs